model/src/main/resources/reference.conf

```diff
@@ -81,6 +81,22 @@ ort {
         commandLine = --copyright --license --info --strip-root --timeout 300
         parseLicenseExpressions = true
       }
+
+      FossId {
+        serverUrl = "https://fossid.example.com/instance/"
+        user = "user"
+        apiKey = "XYZ"
+
+        namingProjectPattern = "$Var1_$Var2"
+        namingScanPattern = "$Var1_#projectBaseCode_$Var3"
+        namingVar1 = "myOrg"
```

Comment:
Is this example correct? I would have expected that the variable must be named `namingVariable1`.

Reply:
No, all variables are prefixed with `naming` but the prefix is not in the variable name.
I did so to be able to group everything naming convention-related in the `ort.conf`, since it's no possible to have structured data such as:
```
        namingConvention {
          projectPattern = "$Var1_$Var2"
          scanPattern = "$Var1_#projectBaseCode_$Var3"
          variables {
            Var = "myOrg"
          }
        }
```

Reply:
Ok, got this; but from my understanding, the constant `NAMING_CONVENTION_VARIABLE_PREFIX` has the value "namingVariable", and you filter for keys starting with this value. So the key `namingVar1` used in the example would not be matched - or am I wrong?

Reply:
You are of course right. Fixed !